### DISCUSSION TITLE
Bypass automatic glyph conversions in Nerd Fonts?

### DISCUSSION THREAD

**Discussion by pcgeek86** [ryanoasis/nerd-fonts#807] [Q&A] [answered]
In Draw.io, when I type "ConfigMap" using the Ubuntu Nerd Font, I get a phone glyph instead of the text `fi`.

Is there a way to "bypass" glyphs, so that I get the literal text `ConfigMap` instead? This might occur with other letter combinations, too, but I only noticed this one so far.

![image](https://user-images.githubusercontent.com/466713/157922070-f7c68ad0-c031-4b87-a8fc-f8e2373c1db8.png)

**Comment by Finii** [comment, marked as answer]
The patched font you use has a defect. This has been fixed (#509 -> #711).
Please use the v2.2.0 RC release.

**Reply by pcgeek86** [reply]
Oh cool, thanks dude!
